rpc/json-rpc/src/id.rs

```diff
@@ -182,20 +182,20 @@ macro_rules! impl_u {
         $(
             impl From<$u> for Id {
                 fn from(u: $u) -> Self {
-                    Self::Num(u as u64)
+                    Self::Num(u64::from(u))
```

Comment:
Oops, original impl took `i8-i64` and `f32/f64` as input which is why `as` was used.